docs/source/rules.rst

```diff
@@ -146,13 +162,60 @@ Example function::
         """
         return any(fnmatch(data, x) for x in whitelist)
 
-Example use of that function within a rule::
+Example usage of the function above in a rule::
+
+    # rules/prod.py
+    
+    from rules.helpers.base import in_set
 
-    @rule(...)
-    def foobar(record):
-        user = 'joe'
-        user_whitelist = { 'mike', 'jin', 'jack', 'mary' }
+    @rule(logs=['example'],
+          outputs=['slack'])
+    def example_rule(record):
+        user = record['user']
+        user_whitelist = {
+          'mike',
+          'jin',
+          'jack',
+          'mary' 
+        }
 
         return in_set(user, user_whitelist)
 
-For information on how to create and run tests to validate rules, see `Rule Testing <rule-testing.html>`_.
\ No newline at end of file
+
+Disabling Rules
+---------------
+
+If a rule needs to be excluded from a rule file without deleting it, a special decorator ``@disable`` can be used.
```

Comment:
I'd explain the use-case. Example: "In the event that you need to disable a rule, due to poor fidelity or any other reason, ...This allows you to keep the rule definition and tests in place, instead of having to remove them entirely"